src/plugins/mer/merhardwaredevicewizardpages.cpp

```diff
@@ -201,14 +208,14 @@ Abi::Architecture MerHardwareDeviceWizardSelectionPage::detectArchitecture(
             || runner.processExitStatus() != SshRemoteProcess::NormalExit
             || runner.processExitCode() != 0) {
         *ok = false;
-        qCWarning(Log::mer) << "Failed to execute uname on target";
+        *error = tr("Failed to detect architecture: Command 'uname' could not be executed");
         return Abi::UnknownArchitecture;
     }
 
     const QString output = QString::fromLatin1(runner.readAllStandardOutput()).trimmed();
     if (output.isEmpty()) {
         *ok = false;
-        qCWarning(Log::mer) << "Empty output from uname executed on target";
+        *error = tr("Failed to detect architecture: Empty output from uname");
```

Comment:
I would prefer "...from the 'uname' command" or so. Similar applies on other places.